lua_modules/ds3231/ds3231.lua

```diff
@@ -5,6 +5,8 @@
 -- Tobie Booth <tbooth@hindbra.in>
 --------------------------------------------------------------------------------
 
+require("bit")
```

Comment:
That confuses me. Wouldn't search for `bit.lua` in the path? And if you need/want to require `bit` wouldn't you have to require `i2c` and any other NodeMCU module used in `ds3231.lua` as well?